inviteme/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/gorilla/schema"
+	"net/http"
+	"net/url"
+)
+
+const (
+	inviteLink    string = "https://jdsb-test-ground-slack-invite.herokuapp.com/"
+	imageLink     string = "http://giphygifs.s3.amazonaws.com/media/5q7XU3MurT0pW/giphy.gif"
+	imageText     string = "I came in like a wrecking ball"
+	imageAltText  string = "Picture of a cat wrecking neatly stacked books"
+	linkMsg       string = "Click on the link below to get an invite to *jdsb-wrecking-ball* workspace: "
```

Comment:
The type declarations here are redundant. The compiler already knows the type to initialize to based on the type of the value.